app/assets/stylesheets/moderators.scss

```diff
@@ -41,6 +41,15 @@
         border-width: 1px 0;
         display: grid;
         grid-template-columns: 4fr 2fr 1fr;
+        .article-title {
+          color: var(--base-90);
+          min-width: 0;
+          header > h3 {
+            text-overflow: ellipsis;
+            overflow: hidden;
+            white-space: nowrap;
+          }
+        }
```

Comment:
> Articles with long titles wrapped around (per ModCenter specs, long titles should be truncated)

Hmmm, is this a good idea? I mean, why would we want to truncate titles in the first place?

Also, since this mod center table has some responsiveness issues, the title area can be very short so the truncated title would be very short too...

Reply:
The truncation was being done in JS before. I suggested if you're going to truncate to do it in CSS.

Reply:
Fair. Still I have doubts if truncation is needed at all. cc @lisasy

Reply:
I see if your point about not truncating. I think it looks okay not truncated:

![Screen Shot 2020-06-25 at 11 49 20 AM](https://user-images.githubusercontent.com/17884966/85753114-f69f1280-b6d9-11ea-8168-798b1d85cf9e.png)

Reply:
Yeah wrapping seems fine. I thought it was being cut off originally.